tests/Auth/AuthAccessGateTest.php

```diff
@@ -12,6 +12,8 @@
 use PHPUnit\Framework\TestCase;
 use stdClass;
 
+include_once 'Enums.php';
```

Comment:
Why not use an import instead?

```php
use Illuminate\Tests\Auth\AbilitiesEnum;
```

I didn't find any usage of `include`/`require` to import classes/interfaces/traits/enums in Laravel.

Reply:
@rodrigopedra I did it like that because its the same way used in other tests in Laravel:

<img width="809" alt="image" src="https://github.com/user-attachments/assets/43f954b8-e05c-4ffd-917b-d9f190b3c42f">

Reply:
Ah ok, I searched on a local project's `vendor` folder, and they don't include the test sources. My bad, sorry =)